tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest>=6.2.2
-pytest<7.0.0
\ No newline at end of file
+pytest<7.0.0
+xmltodict>=0.12.0

```

Comment:
Should not be necessary, might indicate a bug in the action?

Reply:
python -m pip install --upgrade pip
        pip install git+https://github.com/PrediktorAS/opcua-tools.git
        pip install -r tests/requirements.txt

Reply:
This strategy obviously does not work for branches.. should fix this... pip install . instead?